packages/lsd/build.sh

```diff
@@ -1,35 +1,21 @@
-TERMUX_PKG_HOMEPAGE=https://github.com/lsd-rs/lsd 
+TERMUX_PKG_HOMEPAGE=https://github.com/lsd-rs/lsd
 TERMUX_PKG_DESCRIPTION="Next gen ls command"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="Krishna Kanhaiya @kcubeterm"
 TERMUX_PKG_VERSION="1.1.2"
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=https://github.com/lsd-rs/lsd/archive/v${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=cd80dae9a8f6c4c2061f79084468ea6e04c372e932e3712a165119417960e14e
 TERMUX_PKG_DEPENDS="zlib"
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
 
 termux_step_pre_configure() {
-	termux_setup_rust
-
-	mv $TERMUX_PREFIX/lib/libz.so.1{,.tmp}
-	mv $TERMUX_PREFIX/lib/libz.so{,.tmp}
-
-	local _CARGO_TARGET_LIBDIR="target/${CARGO_TARGET_NAME}/release/deps"
-	mkdir -p "${_CARGO_TARGET_LIBDIR}"
-
-	ln -sfT "$(readlink -f "${TERMUX_PREFIX}"/lib/libz.so.1.tmp)" \
```

Comment:
This I'd like to have tested on device before merging.

Reply:
May the device be with you and powered on :upside_down_face: